RealLifeGTAV/dist/metadata.json

```diff
@@ -13,9 +13,9 @@
     "www.reallifegtav.de"
   ],
   "version": "1.0.0",
-  "logo": "https://reallifegtav.de/images/logo.png",
+  "logo": "https://i.imgur.com/upst6fX.png",
   "thumbnail": "https://i.imgur.com/QjWhm3D.png",
```

Comment:
logo needs to be square and 512x512 just like the one uploaded to the discord app

Reply:
still not 512x512

Reply:
https://prnt.sc/uqsvt0 512x512